dev-support/docker-images/jupyter/build.sh

```diff
@@ -16,7 +16,7 @@
 
 set -euxo pipefail
 
-TF_JUPYTER_IMAGE="apache/submarine:tf2.1.0-jupyter"
+JUPYTER_IMAGE="apache/submarine:jupyter-notebook"
```

Comment:
submarine docker image name format is `apache/submarine:{component-name}-{version}`, like., `apache/submarine:mini-0.5.0-SNAPSHOT`
please view : https://hub.docker.com/r/apache/submarine/tags
So I think the `JUPYTER_IMAGE` name should be `apache/submarine:jupyter-notebook-0.5.0-SNAPSHOT`

@pingsutw can you help @lowc1012 push `JUPYTER_IMAGE` push to docker hub repo?

Reply:
I just pushed the image to https://hub.docker.com/r/apache/submarine/tags